pom.xml

```diff
@@ -95,17 +142,6 @@
             </resource>
         </resources>
         <plugins>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-surefire-plugin</artifactId>
-                <version>2.22.0</version>
```

Comment:
Dropping this `<version>` tag is what broke tests running in the `ubuntu-latest` environment. Bummer I didn't notice this 😢 

Will be fixed in #45